go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -359,7 +475,7 @@ func commandShow(cmd *cobra.Command, args []string) error {
 }
 
 func registerCommands(root *cobra.Command) {
-	base.PersistentFlags().StringVar(&baseOptions.Name, "name", "", "The name of the Lookup Vindex to create. This will also be the name of the VReplication workflow created to backfill the Lookup Vindex.")
+	base.PersistentFlags().StringVar(&baseOptions.Name, "name", "", "The name of the Lookup Vindex to create. This will also be the name of the VReplication workflow created to backfill the Lookup Vindex. This will be used only for the workflow name if params-file is used.")
```

Comment:
What benefits do you see in:
  1. Having a file option at all
  2. Forcing people to use the file for everything if they use it

I ask because it used to only be a file with the old legacy client and people found it very challenging to get the format correct in the the JSON file. I explicitly moved most things to flags to make this simpler, with only the minimal data remaining as JSON.